app/src/main/res/color/text_color_hint.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Kenapa ga pake style untuk style text hint?

Reply:
what do you mean?

Reply:
Ini itu buat style dri textnya kan? Kenapa ga di style nya Theme

Reply:
gaa itu utk ngatur warna nya doang biar dinamis ada statenya

Reply:
oh ok